composer.json

```diff
@@ -33,15 +33,15 @@
         }
     },
     "require": {
-        "php": ">=5.5.9",
-        "phpseclib/phpseclib": "^2 || ^3"
+        "php": ">=7.1",
+        "phpseclib/phpseclib": "^2.0.47 || ^3.0.36"
```

Comment:
Why this change?

Reply:
Due [this](https://packagist.org/packages/phpseclib/phpseclib/advisories?version=7795840) security advisory

Reply:
I don't think this is about _educating_ but rather making it safe to install the library.

I do see your point about projects being outdated due to other constraints (like being stuck on a previous phpseclib version), but I think it is the library's responsibility to avoid known security vulnerabilities as much as possible.

If you prefer not to go this way, I'll rollback the constraint change.